doc/examples/reconst_mapmri.py

```diff
@@ -32,35 +47,315 @@
+  of a diffusion tensor during both diffusion gradient pulses. RTAP has been
+  related to the apparent axon diameter [Ozarslan2013, Fick2016]_ under several
+  strong hypothesis on the tissue composition and acquisition protocol.
+- Return to plane probability (RTPP) is a directional index that quantifies the
+  probability that a proton will be on the plane perpendicular to the main
+  eigenvector of a diffusion tensor during both gradient pulses. RTPP is
+  related to the length of a pore [Ozarslan2013]_ but in practice should be
+  similar to that of Gaussian diffusion.
+
+It is also possible to estimate the amount of non-Gaussian diffusion in every
+voxel [Ozarslan2013]_. This quantity is estimated through the ratio between
+Gaussian volume (MAPMRI's first basis function) and the non-Gaussian volume
+(all other basis functions) of the fitted signal. For this value to be
+physically meaningful we must use a b-value threshold in the MAPMRI model. This
+threshold makes the scale estimation in MAPMRI only use samples that
+realistically describe Gaussian diffusion, i.e., at low b-values.
+"""
+map_model_both_ng = mapmri.MapmriModel(gtab, radial_order=radial_order,
+                                       laplacian_regularization=True,
+                                       laplacian_weighting=.05,
+                                       positivity_constraint=True,
+                                       bval_threshold=2000)
+
+mapfit_both_ng = map_model_both_ng.fit(data_small)
+
+fig = plt.figure(figsize=(10, 6))
+ax1 = fig.add_subplot(1, 3, 1, title=r'NG')
+ax1.set_axis_off()
+ind = ax1.imshow(mapfit_both_ng.ng()[:, 0, :].T, interpolation='nearest',
+                 origin='lower', cmap=plt.cm.gray)
+divider = make_axes_locatable(ax1)
+cax = divider.append_axes("right", size="5%", pad=0.05)
+plt.colorbar(ind, cax=cax)
+
+
+ax2 = fig.add_subplot(1, 3, 2, title=r'NG perpendicular')
+ax2.set_axis_off()
+ind = ax2.imshow(mapfit_both_ng.ng_perpendicular()[:, 0, :].T,
+                 interpolation='nearest',
+                 origin='lower', cmap=plt.cm.gray)
+divider = make_axes_locatable(ax2)
+cax = divider.append_axes("right", size="5%", pad=0.05)
+plt.colorbar(ind, cax=cax)
+
+ax3 = fig.add_subplot(1, 3, 3, title=r'NG parallel')
+ax3.set_axis_off()
+ind = ax3.imshow(mapfit_both_ng.ng_parallel()[:, 0, :].T,
+                 interpolation='nearest',
+                 origin='lower', cmap=plt.cm.gray)
+divider = make_axes_locatable(ax3)
+cax = divider.append_axes("right", size="5%", pad=0.05)
+plt.colorbar(ind, cax=cax)
+plt.savefig('MAPMRI_ng.png')
+
+"""
+.. figure:: MAPMRI_ng.png
+   :align: center
+
+On the left we see the overall NG and on the right the directional
+perpendicular NG and parallel NG. The NG ranges from 1 (completely
+non-Gaussian) to 0 (completely Gaussian). The overall NG of a voxel is always
+higher or equal than each of its components. It can be seen that NG has low
+values in the CSF and higher in the white matter.
```

Comment:
Would be interesting to compare NG and MK. Should be really easy to do once this is merged!

cc: @RafaelNH